docs/developer-guide/ksqldb-clients/java-client.md

```diff
@@ -115,7 +115,7 @@ Start by creating a `pom.xml` for your Java application:
       If you’re using ksqlDB for Confluent Platform (CP), use the CP-specific modules
       from [http://packages.confluent.io/maven/](http://packages.confluent.io/maven/)
       by replacing the repositories in the example POM above with a repository with this
-      URL instead. Also update `ksqldb.version` to be a CP version, such as `6.0.0`, instead.
+      URL instead. Also update `ksqldb.version` to be a CP version, such as `{{ site.cprelease }}`, instead.
```

Comment:
Hey @JimGalasyn can you sanity-check me on on two things here?
1. The `cprelease` var is not bumped until we're about ready to release the next CP version. I notice it's currently set to `6.1.0` even though CP 6.1.0 has yet to be released.
2. The variable substitution is fine in code-formatting.

Reply:
Sure, for 1, I try to stay ahead of the next release, to save myself a cherry-pick after the branch is cut. For 2, I built your branch, and it does work! TIL

Reply:
> for 1, I try to stay ahead of the next release, to save myself a cherry-pick after the branch is cut.

As long as we're not so far ahead that the live docs reference a CP version that doesn't exist yet, then this sounds great :) 

> For 2, I built your branch, and it does work! TIL

Thanks, Jim!